src/test/suite/inferFullTargets.test.ts

```diff
@@ -3,7 +3,7 @@ import * as assert from "assert";
 import { inferFullTargets } from "../../inferFullTargets";
 import fixture from "./fixtures/inferFullTargets.fixture";
 
-suite("inferFullTargets", () => {
+suite.skip("inferFullTargets", () => {
```

Comment:
We should probably put a big comment above this so that people noticed that it's skipped